packages/opentelemetry-basic-tracer/src/BasicTracer.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+import { ScopeManager } from '@opentelemetry/scope-base';
+import {
+  BinaryTraceContext,
+  HttpTraceContext,
+  NoopLogger,
+  NoopSpan,
+  randomSpanId,
+  randomTraceId,
+  NEVER_SAMPLER,
+} from '@opentelemetry/core';
+import { BasicTracerConfig } from '../src/types';
+import { BinaryFormat, HttpTextFormat } from '@opentelemetry/types';
+
+/**
+ * This class represents a basic tracer.
+ */
+export class BasicTracer implements types.Tracer {
+  static INVALID_ID = '0';
+
+  private _defaultAttributes: types.Attributes;
+  private _logger: types.Logger;
+  private _binaryFormat: types.BinaryFormat;
+  private _httpTextFormat: types.HttpTextFormat;
+  private _sampler: types.Sampler;
+  private _scopeManager: ScopeManager;
+
+  // TODO: consume invalid span context from `SpanContext.INVALID`
```

Comment:
Could you please use `INVALID_SPAN_CONTEXT` from https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-core/src/trace/spancontext-utils.ts#L21, then we can safely remove `static INVALID_ID = '0';` from top.

Reply:
Thank you, changed!